### PR TITLE
feature: Provide command to clear downloaded copybook cache.

### PR DESCRIPTION
**Pull request by ap891843**
Provide command to clear downloaded copybook cache.

Signed-off-by: ap891843 <aman.prashant@broadcom.com>

### CODE REVIEW

File: clients/cobol-lsp-vscode-extension/package.json [hunk 1]
```diff
@@ -64,6 +64,10 @@
             {
                 "command": "cobol-lsp.commentLine.uncomment",
                 "title": "Remove COBOL Line Comment"
+            },
+            {
+                "command": "cobol-lsp.clear.copybooks.cache",
```
Comment (ap891843) [inline clients/cobol-lsp-vscode-extension/package.json:69]: @zeibura Is this command ok ?

This command clears all the downloaded copybook from .c4z/copybooks

File: clients/cobol-lsp-vscode-extension/src/constants.ts [hunk 2]
```diff
@@ -43,9 +43,20 @@ export const COPYBOOK_CPY_EXT = ".CPY";
 export const COPYBOOK_EXT_ARRAY = [COPYBOOK_CPY_EXT];
 export const EXTENSION_ID = "broadcommfd.cobol-language-support";
 export const TELEMETRY_DEFAULT_CONTENT = "INVALID_INSTRUMENTATION_KEY";
-export const ZOWE_EXT_MISSING_MSG = "Zowe Explorer version 1.15.0 or higher is required to download copybooks from the mainframe."
+export const ZOWE_EXT_MISSING_MSG = "Zowe Explorer version 1.15.0 or higher is required to download copybooks from the mainframe.";
 export const INSTALL_ZOWE = "Install Zowe Explorer";
+export const CLEARING_COPYBOOK_CACHE = "Clearing downloaded copybooks cache";
```
Comment (ap891843) [inline clients/cobol-lsp-vscode-extension/src/constants.ts:48]: @zeibura please suggest a message.

File: clients/cobol-lsp-vscode-extension/package.json (hunk 1 above)
Comment (ap891843) [reply clients/cobol-lsp-vscode-extension/package.json:69]: Yes, F1 menu

File: clients/cobol-lsp-vscode-extension/src/constants.ts (hunk 2 above)
Comment (ap891843) [reply clients/cobol-lsp-vscode-extension/src/constants.ts]: updated

File: clients/cobol-lsp-vscode-extension/package.json (hunk 1 above)
Comment (ap891843) [reply clients/cobol-lsp-vscode-extension/package.json]: Updated to
```
{
                "command": "cobol-lsp.clear.downloaded.copybooks",
                "title": "clear downloaded copybooks"
            }
```

File: clients/cobol-lsp-vscode-extension/package.json [hunk 3]
```diff
@@ -64,6 +64,10 @@
                 "command": "cobol-lsp.commentLine.uncomment",
                 "title": "Remove COBOL Line Comment"
             },
+            {
+                "command": "cobol-lsp.clear.downloaded.copybooks",
+                "title": "clear downloaded copybooks"
```
Comment (niharikasw91) [inline clients/cobol-lsp-vscode-extension/package.json:69]: Suggestion "Clear" ? in title case

**Review by ishche — APPROVED** [review]

File: clients/cobol-lsp-vscode-extension/package.json (hunk 3 above)
Comment (ap891843) [reply clients/cobol-lsp-vscode-extension/package.json]: updated

**Review by niharikasw91 — APPROVED** [review]
LGTM!!

**Pull request merged**